nautobot-app-chatops/{{ cookiecutter.project_slug }}/pyproject.toml

```diff
@@ -23,6 +23,8 @@ packages = [
     { include = "{{ cookiecutter.app_name }}" },
 ]
 include = [
+    "LICENSE",
+    "README.md",
```

Comment:
As I believe I've mentioned elsewhere, I'd recommend against this packaging pattern, as installing the App that includes these files will result in these files being created in the root of the Python `site-packages` directory (rather than within the App's specific subdirectory therein) and potentially causing clashes between packages as a result.

Reply:
Fixed, thanks.